_includes/footer_custom.html

```diff
@@ -79,7 +79,7 @@
     </p>
       <a
         target="_blank"
-        href="https://github.com/machinetranslate/machinetranslate.org/blob/master/{{ page.path }}">
+        href="https://github.com/machinetranslate/machinetranslate.org/blob/master/{% if page.autogenerated %}_data/{{ page.parent | replace: ' ', '_' | downcase }}.json{% else %}{{ page.path }}{% endif %}">
```

Comment:
I bet that this will break, sooner or later.

It's probably simpler to just put something like this in frontmatter:

`autogenerated_from: languages.json`

This could be instead of `autogenerated: true`, or just under it.

Reply:
I've added ```autogenerated_from: languages.json``` under ```autogenerated: true```